packages/components/src/components/flyout/flyout.tsx

```diff
@@ -183,7 +189,7 @@ export class Flyout {
         onTransitionEnd={(e) => onTransitionEnd(e, this.open, this.motionVisibleEnd, this.motionHiddenEnd, this.dialog)}
         {...parseAndGetAriaAttributes({
           'aria-modal': true,
-          'aria-hidden': !this.open,
+          'aria-label': this.ariaLabelText,
```

Comment:
Something like this would be easier than the extra `ariaLabelText` and `headerText`

```suggestion
          ...(hasHeader && { 'aria-label': getSlotTextContent(this.host, 'header') })
```

Reply:
fixed